### PR TITLE
Bug 1609089 Add instuctions to set fluentd label during scaleup

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1609089

### CODE REVIEW

File: install_config/adding_hosts_to_existing_cluster.adoc [hunk 1]
```diff
@@ -61,6 +61,12 @@ etcd3.example.com
 $ ansible-playbook [-i /path/to/file] \
```
Comment (bmcelvee) [inline install_config/adding_hosts_to_existing_cluster.adoc]: The example in the BZ was for `openshift-node`, but this was a close as I could find this this topic.

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL :fallen_leaf:

File: install_config/adding_hosts_to_existing_cluster.adoc [hunk 2]
```diff
@@ -61,6 +61,12 @@ etcd3.example.com
...
   /usr/share/ansible/openshift-ansible/playbooks/openshift-etcd/scaleup.yml
 ----
++
+.. Set the node label to `logging-infra-fluentd: "true"`.
++
```
Comment (mburke5678) [inline install_config/adding_hosts_to_existing_cluster.adoc]: @bmcelvee Should this be Step 3, rather than 2i? Seems odd having only one sub-step.

**Comment by mburke5678** [comment]
@bmcelvee One nit. Otherwise LGTM.

**Review by anpingli — APPROVED** [review]

File: install_config/adding_hosts_to_existing_cluster.adoc (hunk 2 above)
Comment (bmcelvee) [reply install_config/adding_hosts_to_existing_cluster.adoc]: Sure! Updated. :maple_leaf: Thanks!

**Pull request merged**